From edf22f7d6c190010c2da3a26e14e9731775ec5e3 Mon Sep 17 00:00:00 2001
From: Earnin App <verification@account.earnin.com>
Date: Thu, 26 Mar 2020 17:44:07 -0500
Subject: [PATCH] README

Converting
---
 README | 22 ++++++++++++++++++++++
 1 file changed, 22 insertions(+)
 create mode 100644 README

diff --git a/README b/README
new file mode 100644
index 0000000..0cee597
--- /dev/null
+++ b/README
@@ -0,0 +1,22 @@
+    s Form extends Component {
+
+  render() {
+
+    return (
+      <div>
+        <div id="apply_form">
+          <a name="form" id="formAnchor"></a>
+          <script type="text/javascript" src="https://fs22.formsite.com/include/form/embedManager.js?1605047845"></script>
+          <script type="text/javascript">
+          EmbedManager.embed({
+            key: "key",
+            width: "100%",
+            mobileResponsive: true
+          });
+          </script>
+        </div>
+      </div>
+    );
+  }
+}
+Converting coins
\ No newline at end of file
--
Working Copy 4.1.4

